pandas/core/arrays/period.py

```diff
@@ -642,17 +642,17 @@ def _format_native_types(
         values = self.astype(object)
 
         if date_format:
-            formatter = lambda dt: dt.strftime(date_format)
+            formatter = lambda p: p.strftime(date_format)
         else:
-            formatter = lambda dt: str(dt)
+            formatter = lambda p: str(p)
 
         if self._hasna:
             mask = self._isnan
             values[mask] = na_rep
             imask = ~mask
-            values[imask] = np.array([formatter(dt) for dt in values[imask]])
+            values[imask] = np.array([formatter(p) for p in values[imask]])
         else:
-            values = np.array([formatter(dt) for dt in values])
+            values = np.array([formatter(p) for p in values])
```

Comment:
This is just for readability since values are periods and not datetimes here

Reply:
nitpick: can you use "per" or something else thats not 1-character